BepInEx.Core/Bootstrap/BaseChainloader.cs

```diff
@@ -25,14 +25,14 @@ public abstract class BaseChainloader<TPlugin>
     /// <param name="type">Type definition to analyze.</param>
     /// <param name="assemblyLocation">The filepath of the assembly, to keep as metadata.</param>
     /// <returns>If the type represent a valid plugin, returns a <see cref="PluginInfo" /> instance. Otherwise, return null.</returns>
-    public static PluginInfo ToPluginInfo(TypeDefinition type, string assemblyLocation)
+    public static PluginInfo ToPluginInfo<T>(TypeDefinition type, IPluginLoader loader, string assemblyLocation)
```

Comment:
There isn't. I would have to create one that each plugin type implements, but I concluded it could go in my pr that plans to cleanup the existing api anyway

Reply:
Understandable.